supersid/config.py

```diff
@@ -410,35 +422,31 @@ def supersid_check(self):
 
 
 def readConfig(cfg_filename):
-    """
-    read and return the configuration or terminate the program
-    """
-    cfg = Config(cfg_filename)
-    cfg.supersid_check()
-    if cfg.config_ok:
-        assert(1 == len(cfg.filenames))
-        print("Config file '{}' read successfully".format(cfg.filenames[0]))
+    """Read and return the configuration or terminate the program."""
+    config = Config(cfg_filename)
+    config.supersid_check()
+    if config.config_ok:
+        assert len(cfg.filenames) == 1
```

Comment:
`assert len(cfg.filenames) == 1`
should read as
`assert len(config.filenames) == 1`

Reply:
Good catch